test/distributions/test_distributions.py

```diff
@@ -494,6 +494,10 @@ def is_all_nan(tensor):
             'covariance_matrix': torch.tensor([[5.0, -0.5], [-0.5, 1.5]]),
             'df': torch.tensor([2.0]),
         },
+        {
+            'covariance_matrix': torch.tensor([[5.0, -0.5], [-0.5, 1.5]]),
+            'df': torch.tensor(2.0),
```

Comment:
Let us change this to `'df': 2.0` since the function should internally construct a scalar tensor.

Reply:
Thanks for checking out. Reflected it.